test/init.js

```diff
@@ -7,22 +7,23 @@
 
 module.exports = require('should');
 
-var Connector = require('../');
-var DataSource = require('loopback-datasource-juggler').DataSource;
+const Connector = require('../');
+const juggler = require('loopback-datasource-juggler');
+let DataSource = juggler.DataSource;
 
 /** these are the env variables in jenkins **/
 if (process.env.CI && process.env.PACKAGE_NAME &&
   (process.env.BUILD_NUMBER || process.env.BUILD_ID) &&
   (process.env.nodeVersion || process.env.node)) {
-  var buildName = process.env.PACKAGE_NAME.split('-')[2].toUpperCase();
-  var buildNumber = process.env.BUILD_NUMBER || process.env.BUILD_ID;
-  var nodeVersion = process.env.nodeVersion || process.env.node;
-  var os = process.env.OS || process.platform;
-  var schemaName = 'SCHEMA' + buildNumber +
+  const buildName = process.env.PACKAGE_NAME.split('-')[2].toUpperCase();
+  const buildNumber = process.env.BUILD_NUMBER || process.env.BUILD_ID;
+  const nodeVersion = process.env.nodeVersion || process.env.node;
+  const os = process.env.OS || process.platform;
+  const schemaName = 'SCHEMA' + buildNumber +
```

Comment:
I think CI failed because of this. 
CI uses an existing instance of DB2, instead of creating a docker image every time. You might want to create that schemaName in that database. 
@emonddr might be able to help.

Reply:
Both db2 and dashdb (aka db2 warehouse)  loopback connectors require certain environment variables to be set up for tests to run properly.

I wrote something to help people understand how to run the dashdb connector tests from the laptop successfully (in the same way it is run in CI) . See : 

https://github.com/strongloop/loopback-connector-dashdb#running-tests